app/controllers/ingredients_controller.rb

```diff
@@ -0,0 +1,10 @@
+class IngredientsController < ApplicationController
+
+  def index
+    @ingredients = Ingredient.all
```

Comment:
Using the ingredients controller like you have here is reasonable, but another alternative could be to create a `chefs` directory in your controllers folder and adding the nested ingredients controller there. Then, this route would go to `"chefs/ingredients#index"` so you would need to add the `controller: "chefs/ingredients"` parameter to the corresponding route in routes.rb. Either this or your implementation are okay for this IC. But on a project, if you needed to have an ingredients controller that showed ALL ingredients, you would want to use "ingredients#index" for that so this US for a chef's ingredients could go to `"chefs/ingredients#index"`. Then, you wouldn't need this conditional around the chef_id either

Let me know if any of this is unclear!